src/vs/workbench/parts/debug/browser/debugEditorModelManager.ts

```diff
@@ -325,7 +325,7 @@ export class DebugEditorModelManager implements wbext.IWorkbenchContribution {
 
 	private static FOCUSED_STACK_FRAME_DECORATION: editorcommon.IModelDecorationOptions = {
 		isWholeLine: true,
-		className: 'debug-focussed-stack-frame-line',
+		className: 'debug-focused-stack-frame-line',
```

Comment:
(looks good, note that I didn't check for uses/typos outside of these 2 folders, let me know if you notice potential impact)